examples/configs/autoalign/autoalign_factcheck_config/config.yml

```diff
@@ -1,12 +1,19 @@
 models:
   - type: main
     engine: openai
-    model: gpt-3.5-turbo
+    model: gpt-3.5-turbo-instruct
 rails:
-  config:
-    autoalign:
-      parameters:
-        fact_check_endpoint: "https://<AUTOALIGN_ENDPOINT>/factcheck"
-  output:
-    flows:
-      - autoalign factcheck output
+    config:
+        autoalign:
+            parameters:
+                fact_check_endpoint: "https://nvidia.autoalign.ai/factcheck"
```

Comment:
this should be same format as before "https://<AUTOALIGN_ENDPOINT>/factcheck"

Reply:
Will do this once testing is done

Reply:
please change before merging